src/index.js

```diff
@@ -214,12 +223,41 @@ export default class RNPickerSelect extends PureComponent {
         return {};
     }
 
+    scrollToInput() {
+        if (
+            this.props.scrollViewRef == null ||
+            this.props.scrollViewContentOffsetY == null ||
+            this.inputRef == null
+        ) {
+            return;
+        }
+
+        new Promise((resolve) => {
+            this.inputRef.measureInWindow((_x, y, _width, height) => {
+                resolve({ y, height });
+            });
+        }).then((textInputMeasurements) => {
+            // Bottom y-position of TextInput on screen
```

Comment:
What do you think about moving all this logic inside the `measureInWindow()` callback? I think that would simplify the code a bit by removing the Promise.